dash/src/lib/components/DashSpreadGrid.jsx

```diff
@@ -25,6 +25,8 @@ function useResolvedFormatting(formatting) {
                 mappedRule.value = eval(`(${context})=> (${rule.value})`);
             if ('text' in rule)
                 mappedRule.text = eval(`(${context}) => (${rule.text})`);
+            if ('font' in rule)
+                mappedRule.font = isString(rule.font) ? eval(`(${context}) => (${rule.font})`) : rule.font;
```

Comment:
I think for dash we should always evaluate it. This second condition should never be hit (as font is a string by default).

The example in the docs should be adjusted accordingly:
```
"font": "'bold 14px Comic Sans MS'",
```